### PR TITLE
Add doc for volume size label

### LINKED ISSUE #872: Add test cases / documentation for setting PVC volume size
Tracker for https://github.com/kubernetes/kompose/pull/867

We need to:

- [ ] Add documentation for the new label
- [ ] Add tests

### PR DESCRIPTION
**Pull request by hangyan**
With this should close #872

### CODE REVIEW

File: docs/user-guide.md [hunk 1]
```diff
@@ -335,13 +336,31 @@ services:
...
 The currently supported options are:
 
 | Key                  | Value                               |
 |----------------------|-------------------------------------|
 | kompose.service.type | nodeport / clusterip / loadbalancer |
 | kompose.service.expose | true / hostname |
 | kompose.service.expose.tls-secret | secret name |
+| kompose.volume.size | kubernetes supported volume size |
 
 **Note**: `kompose.service.type` label should be defined with `ports` only, otherwise `kompose` will fail.
```
Comment (abitrolly) [inline docs/user-guide.md]: This table looks good at the top of paragraph.

File: docs/user-guide.md [hunk 2]
```diff
@@ -290,9 +290,10 @@ The chart structure is aimed at providing a skeleton for building your Helm char
 
 ## Labels
 
-`kompose` supports Kompose-specific labels within the `docker-compose.yml` file in order to explicitly define a service's behavior upon conversion.
+`kompose` supports Kompose-specific labels within the `docker-compose.yml` file in order to
+explicitly define the generated resources' behavior upon conversion, like Service, PersistentVolumeClaim...
```
Comment (abitrolly) [inline docs/user-guide.md]: `in order` can be omitted to simplify the text.

**Comment by cdrage** [comment]
What @abitrolly said ^^. Other than those two comments. LGTM.

**Comment by hangyan** [comment]
@cdrage @abitrolly updated, thanks!

**Comment by cdrage** [comment]
LGTM! :+1:

**Pull request merged**